foundation_cms/core/views.py

```diff
@@ -0,0 +1,23 @@
+from django.shortcuts import render
+from foundation_cms.base.models import AbstractArticlePage
+
+
+def listing_page(request):
+    # Get the tag from the query parameters
+    tag = request.GET.get('tag')
+
+    # Get all pages
+    pages = AbstractArticlePage.objects.live().public()
+
+    # Filter by tag if provided
+    if tag:
+        articles = pages.filter(base_tagged_items__tag__slug=tag)
+    else:
+        articles = pages.all()
+
+    context = {
+        'articles': articles,
+        'active_tag': tag if tag else 'Latest Posts'
+    }
+
+    return render(request, "patterns/pages/core/listing_page.html", context)
```

Comment:
Updates Abstract model to Page, adds locale check, and updates variable to `pages` for template.

```suggestion
from wagtail.pages import Locale, Page

def listing_page(request):
    # Get the tag from the query parameters
    tag = request.GET.get('tag')
    
    # Get the current locale
    locale = Locale.get_active()

    # Get all pages for the current locale
    pages = Page.objects.live().public().filter(locale=locale)

    # Filter by tag if provided
    if tag:
        pages = pages.filter(base_tagged_items__tag__slug=tag)
    else:
        pages = pages.all()

    context = {
        'pages': pages,
        'active_tag': tag if tag else 'Latest Posts'
    }

    return render(request, "patterns/pages/core/listing_page.html", context)
```